### ISSUE TITLE
Uncaught Error: Unknown Storeon event Symbol()

### ISSUE THREAD

**Issue by pepijnolivier** [storeon/router#22]
Everytime I click a link in my react app, I'm seeing the following console warning:

```
vendor.js:112479 Uncaught Error: Unknown Storeon event Symbol()
    at vendor.js:112479
    at vendor.js:112565
    at Array.forEach (<anonymous>)
    at dispatch (vendor.js:112564)
    at Object.dispatch (vendor.js:112558)
    at vendor.js:3008
    at vendor.js:112565
    at Array.forEach (<anonymous>)
    at Object.dispatch (vendor.js:112564)
    at HTMLHtmlElement.<anonymous> (vendor.js:3052)
```

Everything in the app works fine, including routing. But every time I click a link, this console error appears. Am I missing something obvious here ? Has anyone experienced this before ?

**Comment by ivan-solovev** [comment]
Hi @pepijnolivier! I tried different ways to reproduce this behavior. I didn't succeed. Can you give a config for Storeon and the router itself?

**Comment by pepijnolivier** [comment]
Hi @isolovev 

Thanks for looking into it! This is how I initialize the app.
Is there a base JSFiddle or Codepen somewhere I can use to try and reproduce it online ?
I might be able to try that next week. Anyways, the webapp still works perfectly, it's just weird this console error is triggered somehow.

```
import React from 'react';
import useStoreon from 'storeon/react';
import StoreContext from 'storeon/react/context';

import router from '@storeon/router';

import MyPageOne from '../pages/MyPageOne';
import MyPageTwo from '../pages/MyPageTwo';
import NotFound from '../pages/NotFound';

import dataStore from '../../store';

function Router() {
  const { [router.key]: route } = useStoreon(router.key);

  dataStore.dispatch('myapp/reset');

  switch (route.match.page) {
    case 'PageOne':
      return <MyPageOne />;

    case 'PageTwo':
      return <MyPageTwo id={route.match.id} />;

    default:
      return <NotFound />;
  }
}

export default function App() {
  return (
    <StoreContext.Provider value={dataStore}>
      <Router />
    </StoreContext.Provider>
  );
}

```


And this is what's inside the store:

```
import createStore from 'storeon';
import router from '@storeon/router';

import stateOne from './stateOne';
import stateTwo from './stateTwo';
import globals from './globals';

export default createStore([
  stateOne,
  stateTwo,
  globals,
  router.createRouter([
    ['/page1', () => ({ page: 'PageOne' })],
    ['/page1/*', id => ({ page: 'PageTwo', id })],
  ]),
  // eslint-disable-next-line global-require
  process.env.NODE_ENV !== 'production' && require('storeon/devtools'),
]);

```

**Comment by ivan-solovev** [comment]
I tried your code. I can't reproduce the problem. Can you try to do it on Github and build the code using Parceljs, so as not to waste time on Webpack configuration? Or using JSFiddle and Codepen.

I want to help with this problem. 😄

**Comment by huhao98** [comment]
The router code uses empty Symbol for `routerChanged`, `routerNavigate` events. The __storeon/devtools__ will throw an error given the event name is empty.

Following are the code in __storeon/devtools__ that cause this problem.

```javascript
 store.on('@dispatch', (state, data) => {
      let event = String(data[0])
      if (event !== 'UPDATE_FROM_DEVTOOLS' && prev !== 'UPDATE_FROM_DEVTOOLS') {
        if (event[0] !== '@' && (!data[2] || data[2].length === 0)) {
          throw new Error('Unknown Storeon event ' + event)
        }
        if (event !== '@changed' || Object.keys(data[1]).length) {
          ReduxTool.send({ type: event, payload: data[1] }, state)
        }
      }
      prev = event
    })
```

**Comment by ivan-solovev** [comment]
@huhao98 thank you for the sample code. I'll check this case today.

**Comment by ivan-solovev** [comment]
@huhao98 @pepijnolivier I fixed this bug.

**Comment by ivan-solovev** [comment]
Released version 1.0.1.
